core/src/plugins/stored/CMakeLists.txt

```diff
@@ -30,11 +30,15 @@ if(HAVE_WIN32)
   add_definitions(-DMS_WIN${WINDOWS_BITS})
 endif()
 
+include_directories(${PROJECT_SOURCE_DIR}/src/fastlz/include)
```

Comment:
We should not introduce more `include_directories`. The right thing would be to add this to target_include_directories of bareosfastlz with the PUBLIC or INTERFACE keywords.
In fact I should have done that when I imported fastlz.